pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2919,6 +2919,9 @@ type CephBlockPoolRadosNamespaceList struct {
 
 // CephBlockPoolRadosNamespaceSpec represents the specification of a CephBlockPool Rados Namespace
 type CephBlockPoolRadosNamespaceSpec struct {
+	// The name of the CephBlockPoolRadosNamespaceSpec namespace. If not set, the default is the name of the CR.
+	// +optional
+	Name string `json:"name,omitempty"`
```

Comment:
Let's add CEL rules to make this immutable. 
https://github.com/csi-addons/kubernetes-csi-addons/blob/main/apis/replication.storage/v1alpha1/volumereplication_types.go#L71

make blockPoolName below too (this can be another pr).

Reply:
ok

Reply:
Note that #13177 needs to merge before we can merge other CEL rules.

Reply:
#13177 merged, want to add the CEL rule now?

Reply:
Will open a separate PR for both svgs and radosnamesapce for adding this

Reply:
And it also need to add cel rules in the pool, which wont be right way to add them

Reply:
Added a tracker https://github.com/rook/rook/issues/13315

Reply:
@travisn